python/docs/source/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,38 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I referred to pandas' https://raw.githubusercontent.com/pandas-dev/pandas/master/doc/_templates/autosummary/class.rst